### PR TITLE
Intuitive drivetrain sim instructions

### PR DESCRIPTION
**Pull request by michaelleong25**
Added the link to the field2d widget page to the note because making the field viewable isn't very obvious.

### CODE REVIEW

File: source/docs/software/wpilib-tools/robot-simulation/drivesim-tutorial/odometry-simgui.rst [hunk 1]
```diff
@@ -73,4 +73,4 @@ Finally, the pose from your odometry must be updated periodically into the ``Fie
 
 .. important:: It is important that this code is placed in a regular ``periodic()`` method -- one that is called periodically regardless of mode of operation. If you are using the command-based library, this method already exists. If not, you are responsible for calling this method periodically from the main ``Robot`` class.
 
-.. note:: At this point we have covered all of the code changes required to run your code.  You should head to the :ref:`Simulation User Interface page <docs/software/wpilib-tools/robot-simulation/simulation-gui:Simulation Specific User Interface Elements>` for more info on how to run the simulation and add the field that your simulated robot will run on to the GUI.
+.. note:: At this point we have covered all of the code changes required to run your code.  You should head to the :ref:`Simulation User Interface page <docs/software/wpilib-tools/robot-simulation/simulation-gui:Simulation Specific User Interface Elements>` for more info on how to run the simulation and the :ref:`Field2d Widget page <docs/software/dashboards/glass/field2d-widgets:The Field2d Widget>` to add the field that your simulated robot will run on to the GUI.
```
Comment (sciencewhiz) [inline source/docs/software/wpilib-tools/robot-simulation/drivesim-tutorial/odometry-simgui.rst:76]:
```suggestion
.. note:: At this point we have covered all of the code changes required to run your code.  You should head to the :ref:`Simulation User Interface page <docs/software/wpilib-tools/robot-simulation/simulation-gui:Simulation Specific User Interface Elements>` for more info on how to run the simulation and the :ref:`Field2d Widget page <docs/software/dashboards/glass/field2d-widget:The Field2d Widget>` to add the field that your simulated robot will run on to the GUI.
```

**Comment by sciencewhiz** [comment]
Maybe the link should go to the "Viewing Trajectories with Glass" heading, since the earlier parts are already covered in this tutorial?

**Review by sciencewhiz — APPROVED** [review]

**Review by sciencewhiz — APPROVED** [review]

**Review by Daltz333 — APPROVED** [review]

**Comment by michaelleong25** [comment]
Is this all good to merge?

**Comment by Daltz333** [comment]
Whoops. Thought I merged this.

**Pull request merged**